### DISCUSSION TITLE
command line

### DISCUSSION THREAD

**Discussion by phil8637** [rern/rAudio#1496] [General]
Hello all
I would like to reload my playlist with a single command line
ex: mpc stop; mpc clear; mpc load "myplaylist"; mpc play
it was ok with runeaudio 0.5b
works more with Raudio1

thank you in advance

**Comment by Maikeleg** [comment]
Hi there,
It still works, I control mpc from another rpi but the command is the same (without -h $ip);
```
 #!/bin/bash
 ip=192.168.88.123
 /usr/bin/mpc -h $ip clear
 /usr/bin/mpc -h $ip stop
```
when i check with
`
/usr/bin/mpc -h $ip current
`
the playlist is removed
It might be that you have execute mpc with sudo
PS: Before you execute 'mpc play' you need to load a playlist
PS2: I'm on rAudio 20231001

**Comment by phil8637** [comment]
hello
thank you great
I made a mistake in my order, everything is working again, thank you for your work
Best regards
